packages/node_modules/@ciscospark/redux-module-avatar/src/actions.js

```diff
@@ -40,9 +40,13 @@ function fetchUserAvatar(userId, spark) {
   return (dispatch) => {
     dispatch(addAvatarBegin(userId));
     return spark.people.get(userId)
-      .then((person) => dispatch(addAvatar(userId, convertToSmallAvatar(person.avatar))))
+      .then((person) => {
+        const avatar = convertToSmallAvatar(person.avatar);
+        dispatch(addAvatar(userId, avatar));
+        return Promise.resolve(avatar);
+      })
       .catch(() => {
-        dispatch(addAvatar(userId));
+        dispatch(addAvatar({id: userId}));
```

Comment:
This doesn't look like it would work. `addAvatar` takes two params.

Reply:
Yea... fixed later. But this catch is actually to just inject an empty placeholder to tell us we've tried to fetch before and to not to try again.